buildSrc/src/main/java/api.kt

```diff
@@ -1,3 +1,51 @@
+import com.stepango.forma.FormaConfiguration
+import com.stepango.forma.Api
+import com.stepango.forma.Validator
+import com.stepango.forma.throwProjectValidationError
 import org.gradle.api.Project
+import org.gradle.kotlin.dsl.apply
 
-fun Project.api(): Unit = TODO()
\ No newline at end of file
+fun Project.api(
+    dependencies: NamedDependency = emptyDependency(),
+    buildConfiguration: BuildConfiguration = emptyDependency(),
+    consumerMinificationFiles: Set<String> = emptySet(),
+    manifestPlaceholders: Map<String, Any> = emptyMap()
+) = api(
+    dependencies,
+    buildConfiguration,
+    consumerMinificationFiles,
+    manifestPlaceholders,
+    Forma.configuration
+)
+
+@Suppress("UnstableApiUsage")
+internal fun Project.api(
+    dependencies: NamedDependency,
+    buildConfiguration: BuildConfiguration,
+    consumerMinificationFiles: Set<String>,
+    manifestPlaceholders: Map<String, Any>,
+    formaConfiguration: FormaConfiguration,
+    validator: Validator = ApiValidator
+) {
+    apply(plugin = "com.android.library")
```

Comment:
We don't need android here, for build perf reasons. 
Also we need `api` and `kt_api` for java and kotlin correspondingly